homeassistant/components/syncthing/__init__.py

```diff
@@ -0,0 +1,171 @@
+"""The syncthing integration."""
+import asyncio
+import logging
+
+import aiosyncthing
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import (
+    CONF_NAME,
+    CONF_TOKEN,
+    CONF_URL,
+    CONF_VERIFY_SSL,
+    EVENT_HOMEASSISTANT_STOP,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+from .const import (
+    DOMAIN,
+    EVENTS,
+    RECONNECT_INTERVAL,
+    SERVER_AVAILABLE,
+    SERVER_UNAVAILABLE,
+)
+
+CONFIG_SCHEMA = vol.Schema({DOMAIN: vol.Schema({})}, extra=vol.ALLOW_EXTRA)
```

Comment:
Since this integration is surely config_entry based I don't think CONFIG_SCHEMA is used for anything.